lib/pages/chat/input_bar.dart

```diff
@@ -461,6 +461,10 @@ class InputBar extends StatelessWidget {
                 final data = content.data;
                 if (data == null) return;
 
+                if (content.mimeType.startsWith("image/")) {
+                  onSubmitImage!(data);
```

Comment:
I use this feature mostly with the GBoard when sending a sticker or GIF. Having a dialog every time would be just overhead IMO

Reply:
Yeah... it will be very inconvenient....

But I think media is a bit expensive for the matrix. Currently both Android and web can upload images by pasting. So without visual confirmation, users might accidentally upload contents either they don't want or in its original quality (average screenshot takes about 7M, and in most case it's unnecessary to keep the original quality). Thus, putting more burden on home-level servers.

What if we only show the dialog for images larger than a certain size (e.g. 1M)?